src/poolCaching/onchainData.ts

```diff
@@ -33,13 +33,6 @@ export async function getOnChainBalances(
     const multiPool = new Multicaller(multiAddress, provider, abis);
 
     subgraphPools.forEach((pool, i) => {
```

Comment:
```suggestion
    subgraphPools.forEach((pool) => {
```
We don't need the index here.